extra_tests/snippets/dir_module/__init__.py

```
@@ -1,2 +1,2 @@
-from .relative import value
 from .dir_module_inner import value2
```

Flake8 (7.3.0)

[error] '.dir_module_inner.value2' imported but unused

(F401)